RealmSwift-swift2.0/Realm.swift

```diff
@@ -279,8 +279,7 @@ public final class Realm {
     - returns: The created object.
     */
     public func create<T: Object>(type: T.Type, value: AnyObject = [:], update: Bool = false) -> T {
-        // FIXME: use T.className()
-        let className = (T.self as Object.Type).className()
+        let className = (type as Object.Type).className()
```

Comment:
Is the `Object.Type` cast still necessary? You removed the fixme, but not the cast...